pkg/models/instances.go

```diff
@@ -201,14 +201,13 @@ type GetInstanceResponseInstanceConfig struct {
 	Createbackup         bool          `json:"createBackup,omitempty"`
 	Memorydisplay        string        `json:"memoryDisplay,omitempty"`
 	Backup               struct {
-		Veeammanagedserver string `json:"veeamManagedServer,omitempty"`
```

Comment:
Any reason for removing this key `Veeammanagedserver` ? I mean do we no longer need it or does it not server any purpose to us

Reply:
Got it! Just saw Reuben's comment. It is not supported.  Hope you have tested it completely with this new key removed!